### PR TITLE
Add installation method module and remove duplicated doc from configuration

### PR DESCRIPTION
**Pull request by thomasnares**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | Add installation methos from console, and remove duplicated doc from configuration

### CODE REVIEW

File: modules/creation/tutorial.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ class MyModule extends Module
 }
 ```
 
-Let's examine each line...
+Let's examine each line :
```
Comment (kpodemski) [inline modules/creation/tutorial.md]:
```suggestion
Let's examine each line:
```

in English we don't need space before `:` 👍

File: modules/creation/tutorial.md [hunk 2]
```diff
@@ -245,49 +245,12 @@ As you can see, our three blocks of code (`__construct()`, `install()` and `unin
 
 This is a PrestaShop-specific object that allows to easily manage all the shop's settings. It stores its data on the `ps_configuration` database table.
```
Comment (kpodemski) [inline modules/creation/tutorial.md]:
```suggestion
This PrestaShop-specific object allows you to easily manage all the shop's settings. It stores its data on the `PREFIX_configuration` database table.
```

I think we can try to use `PREFIX_tablename` as it is recommended to use the custom prefix.

**Review by matks — APPROVED** [review]
Looks good to me 😄 

Just to be sure let's see @kpodemski final review

File: modules/creation/tutorial.md [hunk 3]
```diff
@@ -326,8 +289,32 @@ There are many free icon libraries available on the web. Here are a few:
 
 ## Installing the module
 
+You have two options to install a module : via back office interface, or via Symfony Console
```
Comment (kpodemski) [inline modules/creation/tutorial.md]:
```suggestion
You have two options to install a module: from the back office interface or using the Symfony Console component (CLI-base installation)
```

File: modules/creation/tutorial.md [hunk 4]
```diff
@@ -326,8 +289,32 @@ There are many free icon libraries available on the web. Here are a few:
...
+
+### Install module via back office interface
```
Comment (kpodemski) [inline modules/creation/tutorial.md]:
```suggestion
### Install module from the back office interface
```

File: modules/creation/tutorial.md [hunk 5]
```diff
@@ -326,8 +289,32 @@ There are many free icon libraries available on the web. Here are a few:
...
+
 Now that all basics are in place, reload the back office's "Module Catalog" page, in the "Front office features" section, you should find your module. Install it (or reset it if it is already installed).
 
+### Install module via Symfony Console
```
Comment (kpodemski) [inline modules/creation/tutorial.md]:
```suggestion
### Install module from CLI using the Symfony Console component
```

File: modules/creation/tutorial.md [hunk 6]
```diff
@@ -326,8 +289,32 @@ There are many free icon libraries available on the web. Here are a few:
...
+
+Access your project's directory with a CLI, and run :
```
Comment (kpodemski) [inline modules/creation/tutorial.md]:
```suggestion
Access your project's directory with a CLI, and run:
```

File: modules/creation/tutorial.md [hunk 7]
```diff
@@ -326,8 +289,32 @@ There are many free icon libraries available on the web. Here are a few:
...
+
+```shell
+php bin/console prestashop:module install mymodule
+```
+
+Where `mymodule` is your module's name.
+
+To uninstall module, run :
```
Comment (kpodemski) [inline modules/creation/tutorial.md]:
```suggestion
To uninstall the module, run the following:
```

**Review by kpodemski — CHANGES_REQUESTED** [review]

**Comment by thomasnares** [comment]
Thanks @kpodemski 👍

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Thanks @thomasnares 

Let's backport it to 1.7 if you think it is relevant. Seems like it is, but it's worth checking if options are the same.

**Pull request merged**